drivers/sensor/bmi270/bmi270.c

```diff
@@ -678,6 +684,7 @@ static int bmi270_init(const struct device *dev)
 	if (ret != 0) {
 		return ret;
 	}
+	k_usleep(BMI270_INTER_WRITE_DELAY_US);
```

Comment:
SInce it is done many times, maybe you can directly put it in reg_write().
Does it make sense?

Reply:
Even though it's repeated multiple times, the inter-write delays are not required during initialization (as `bmi270_init()` disables adv_power_save before the initial set of reg-writes and re-enables it at the end).

If we're concerned about the DRY principle, perhaps a helper function like `reg_write_with_delay()` could help us factor them out altogether. Otherwise, I suggest we keep it as it is.